src/components/Badge/Badge.tsx

```diff
@@ -22,14 +23,17 @@ class Badge extends React.Component<BadgeProps> {
   static defaultProps: DefaultProps = { priority: BADGE_PRIORITY.default };
 
   render() {
-    const { priority, children, className } = this.props;
+    const { priority, children, className, icon } = this.props;
     return (
       <div
         data-priority={priority}
         className={st(classes.root, { priority }, className)}
         data-hook={this.props['data-hook']}
       >
-        {children}
+        <div style={{ display: 'flex' }}>
```

Comment:
We prefer to not use `style` attributes in our components.
So please use a class.
btw, is this `div` really needed, or is the root `div` enough?